### ISSUE TITLE
Rounding bug for large files

### ISSUE THREAD

**Issue by draptik** [athityakumar/colorls#600]
Files larger than 1GB are rounded wrong?

Example:

```sh
~/Downloads
❯ colorls -l *img
   rw-r--r--   1   patrick   patrick      1 GiB   Fri Sep 29 20:42:49 2023    2023-05-03-raspios-bullseye-arm64-lite.img
   rw-r--r--   1   patrick   patrick      1 GiB   Fri Sep 29 20:41:50 2023    2023-05-03-raspios-bullseye-armhf-lite.img

~/Downloads
❯ ls -alh *img
-rw-r--r-- 1 patrick patrick 2.0G Sep 29 20:42 2023-05-03-raspios-bullseye-arm64-lite.img
-rw-r--r-- 1 patrick patrick 1.9G Sep 29 20:41 2023-05-03-raspios-bullseye-armhf-lite.img

~/Downloads
❯ ls -al *img
-rw-r--r-- 1 patrick patrick 2101346304 Sep 29 20:42 2023-05-03-raspios-bullseye-arm64-lite.img
-rw-r--r-- 1 patrick patrick 1967128576 Sep 29 20:41 2023-05-03-raspios-bullseye-armhf-lite.img
```
Both files are more 2GB-like, than 1GB-like.

- Type of issue :
  - [x] Bug in existing feature

colorls: 1.4.6
zsh: 5.9
os: arch

**Comment by ayushpoddar** [comment]
I want to pick it up @avdv

**Comment by avdv** [comment]
> I want to pick it up @avdv 

Sure, thank you!

**Comment by ayushpoddar** [comment]
@avdv I have raised the PR

**Comment by avdv** [comment]
Fixed by #622
